src/app/modules/entity/entity-table/entity-table.component.ts

```diff
@@ -1178,6 +1180,10 @@ export class EntityTableComponent<Row extends SomeRow = SomeRow> implements OnIn
     return this.conf.config.multiSelect ? this.currentColumns[1].prop : this.currentColumns[0].prop;
   }
 
+  getRowValue(value: boolean | string): string {
+    return typeof value === 'boolean' ? this.yesNoPipe.transform(value) : value;
```

Comment:
I'm worried about unintended consequences of editing a core component like this.
I'd rather we use local transformations in disks component.

Reply:
@undsoft - you mean modify it from boolean to string in disks component?

Reply:
Yes.

Reply:
We need more tables to be refactored to ix-table2 😄